### ISSUE TITLE
frescobaldi fails to start

### ISSUE THREAD

**Issue by puzzlewolf** [NixOS/nixpkgs#76847]
**Describe the bug**
frescobaldi fails to start.

**To Reproduce**
```
$ nix-shell -p frescobaldi

$ frescobaldi
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.
```

**Expected behavior**
frescobaldi should start

**Additional context**
Maybe related to #53079.
Also, there is a new release of frescobaldi, [3.1](https://github.com/frescobaldi/frescobaldi/tree/v3.1), using QtWebEngine instead of QtWebKit.

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.6, NixOS, 19.09.1686.69ed29f5f41 (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
  - frescobaldi
# a list of nixos modules affected by the problem
module:
```

**Comment by veprbl** [comment]
This is #65399, it needs `wrapQtAppsHook`

**Comment by filalex77** [comment]
Can't reproduce on Sway (Wayland):

```
$ frescobaldi
Using Wayland-EGL
Using the 'xdg-shell' shell integration
# executable runs fine
```

**Comment by puzzlewolf** [comment]
This should be fixed with #76889
